src/Template/Template.WebAPI/Backend/Issues/Issue.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Backend.Contracts.Issues;
+using GoldenEye.Backend.Core.DDD.Events;
+using GoldenEye.Shared.Core.Objects.General;
+
+namespace Backend.Issues
+{
+    public class Issue : IEvent, IHasId
```

Comment:
This class is not an event, it's aggregate, so it shouldn't implement the `IEvent` interface, it should implement `IAggregate` thou